tests/test_schema.py

```diff
@@ -30,13 +30,57 @@ def load_json(filename):
     return doc
 
 
+def test_delivery():
```

Comment:
You're testing two different things here. If the first `assert` fails, the second will never execute so you'll have no way of knowing if it would have passed. These should be separate tests.

Also, is this actually related to commercial model and use types?